ui/main/src/app/services/sound-notification.service.spec.ts

```diff
@@ -109,55 +112,162 @@ describe('SoundNotificationService', () => {
+
+    expect(playSound).toHaveBeenCalledTimes(0);
+    service.handleUpdatedCard(actionCard,UpdateTrigger.REMINDER, [actionCard.id,informationCard.id]);
+    service.handleUpdatedCard(informationCard,UpdateTrigger.REMINDER,[actionCard.id,informationCard.id]);
+    expect(playSound).toHaveBeenCalledTimes(1);
+    expect(playSound).toHaveBeenCalledWith(service.actionSound);
+
+  });
+
+  it('should play appropriate sound for all severities if updated card is a reminder, is visible and all severities are on', () => {
+
+    //Handles 1 card of each severity, should play all four sounds
+
+    configAllSeveritiesOn();
+
+    const today = new Date().getTime();
+    const alarmCard = getOneRandomLightCard({severity:Severity.ALARM, publishDate:today});
+    const actionCard = getOneRandomLightCard({severity:Severity.ACTION, publishDate:today});
+    const compliantCard = getOneRandomLightCard({severity:Severity.COMPLIANT, publishDate:today});
+    const informationCard = getOneRandomLightCard({severity:Severity.INFORMATION, publishDate:today});
+
+    expect(playSound).toHaveBeenCalledTimes(0);
+    service.handleUpdatedCard(alarmCard,UpdateTrigger.REMINDER,[alarmCard.id, actionCard.id, compliantCard.id, informationCard.id]);
+    service.handleUpdatedCard(actionCard,UpdateTrigger.REMINDER,[alarmCard.id, actionCard.id, compliantCard.id, informationCard.id]);
+    service.handleUpdatedCard(compliantCard,UpdateTrigger.REMINDER,[alarmCard.id, actionCard.id, compliantCard.id, informationCard.id]);
+    service.handleUpdatedCard(informationCard,UpdateTrigger.REMINDER,[alarmCard.id, actionCard.id, compliantCard.id, informationCard.id]);
+
+    expect(playSound).toHaveBeenCalledTimes(4);
+    expect(playSound).toHaveBeenCalledWith(service.alarmSound);
+    expect(playSound).toHaveBeenCalledWith(service.actionSound);
+    expect(playSound).toHaveBeenCalledWith(service.compliantSound);
+    expect(playSound).toHaveBeenCalledWith(service.informationSound);
+
+  });
+
+  it('should not play sound if all severities are off even if updated card is a reminder and is visible', () => {
+
+    configAllSeveritiesOff();
+
+    const today = new Date().getTime();
+    const card = getOneRandomLightCard({publishDate:today});
+
+    service.handleUpdatedCard(card,UpdateTrigger.REMINDER,[card.id]);
+    expect(playSound).not.toHaveBeenCalled();
+
+  });
+
+  it('should not play sound if no configuration is made even if updated card is a reminder and is visible', () => {
+
+    noConfig();
+
+    const today = new Date().getTime();
+    const card = getOneRandomLightCard({ publishDate: today});
+
+    service.handleUpdatedCard(card,UpdateTrigger.REMINDER,[card.id]);
+    expect(playSound).not.toHaveBeenCalled();
+
+  });
+
+  /* For these tests, settings configuration property to true for all severities */
+
+  it('should not play sound if updated card is not visible', () => {
```

Comment:
... if udpate card **is reminder and** is not visible

Reply:
You are right, fixed it.